site/components/pages/Buy/index.tsx

```diff
@@ -239,32 +238,41 @@ export const Buy: NextPage<Props> = ({}) => {
               />
             </div>
             <Text t="h3" as="h3" align="center">
-              What do I need to buy KLIMA on Zapper?
+              <Trans id="buy.dex.how_zapper">
+                How do I buy KLIMA on Zapper?
+              </Trans>
             </Text>
           </div>
-          <BuyKlimaTimeline />
+          <ZapperTimeline />
         </div>
       </Section>
       <div ref={advancedSectionRef}></div>
       <Section className={styles.section}>
         <div className={styles.buyContainer}>
           <div className={styles.buy_textGroup}>
-            <Text t="h4" align="center" color="lighter">
-              ADVANCED
+            <Text t="h4" align="center" color="lighter" uppercase>
+              <Trans id="buy.advanced">Advanced</Trans>
             </Text>
             <Text t="h2" as="h2" align="center">
-              Bonding
+              <Trans id="buy.bonding">Bonding</Trans>
+            </Text>
+            <Text t="body3" color="lighter">
+              <Trans id="buy.bonding.description_01">
+                Bonding is the process of depositing carbon assets in exchange
+                for discounted KLIMA. This is one of the ways the Klima Treasury
+                absorbs carbon credits for positive climate impact, and to
+                provide the backing for each KLIMA token.
+              </Trans>
```

Comment:
"This is one of the ways the Klima Treasury absorbs carbon credits for positive climate impact. Bonding also provides backing for each KLIMA token."